src/components/media-views.js

```diff
@@ -758,7 +758,7 @@ AFRAME.registerComponent("media-image", {
         // invert the geometry on the x-axis so that all of the faces point inward
         geometry.scale(-1, 1, 1);
       } else {
-        geometry = new THREE.PlaneGeometry();
+        geometry = new THREE.PlaneBufferGeometry(1, 1, 1, 1, texture.flipY);
```

Comment:
See also:
```
src/react-components/chat-message.js:192:    const geometry = new THREE.PlaneGeometry();
src/components/camera-tool.js:108:      const geometry = new THREE.PlaneGeometry(width, width / this.camera.aspect);
```

Reply:
Oh nice, good catch. I had actually broken the chat messages since they were using texture loaded and not accounting for flipY. Camera tool wasn't broken since it doesn't use a TextureLoader (since this PlaneGeometry is for the preview view). It should be using PlaneBufferGeometry anyway, fixed.